app/apps/users/migrations/0003_contactperson.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.3 on 2020-12-02 14:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0002_auto_20201014_0557"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ContactPerson",
+            fields=[
```

Comment:
Mis je nog een migratie? Ik "ContactPerson" als model naam, maar in models.py zie ik SupportContact. Dus volgens mij is die renaming nog niet meegepakt in een migratie.

Reply:
Woops lijkt erop dat 1 commit niet gepusht is wordt gefixt